web/src/filter.ts

```diff
@@ -453,7 +453,7 @@ export class Filter {
     }
 
     // Parse a string into a list of terms (see below).
-    static parse(str: string): NarrowTerm[] {
+    static parse(str: string, for_pills = false): NarrowTerm[] {
```

Comment:
this could be part of work towards making a separate `SearchPillTerm`

Reply:
Yeah, that makes sense. 

The other medium-term thing I notice reading this is that we've not yet changed the web app to just use user IDs rather than emails for search pills.